docs/pages/quickstart_guide.mdx

```diff
@@ -0,0 +1,86 @@
+```
+
+**Namespace Prefixes:**
+Each origin supports one or more *namespace prefixes*, which are analogous to the folders or directories from your computer that you use to organize files. Below is an example object:
+```bash
+/demo/testfile.txt
+```
+This is a simple example where the namespace prefix would be `/demo` and the object is named `/testfile.txt`.
+
+However, things can get a little tricky when you add a longer namespace prefix and/or a longer filepath within that namespace where the object lives. For example:
+```bash
+/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+In this example, the namespace prefix is `/ospool/uc-shared/public/`, and the actual object is named `/OSG-Staff/validation/test.txt`. While the namespace prefix always comes before the object name, it can get a little confusing once paths and names get longer.
+
+**How to differenciate namespace prefix and object name?**
+To discover where the namespace prefix ends and the object name begins, there are a few options. One way is to access your federation's registry (i.e. [https://osdf-registry.osg-htc.org/view/registry/](https://osdf-registry.osg-htc.org/view/registry/)) which should list the namespaces available in the federation. Another option is also asking your federation administrator for how to access the registry.
+>We are currently working to improve user experience exploring federations which includes better ways to differentiate namespace prefix and object name.
+
+
+## Your First Pelican Client Command
+Accessing objects from a Pelican federation requires the use of the Pelican client commands. Here we will be using the `pelican object get` command to get an object from a federation. This command is formatted like so:
+```console
+pelican object get pelican://<federation-url>/<namespace-prefix>/<path/to/federation/object> </local/path/to/destination>
+```
+We can break down the components of this command before you try it yourself:
+- `pelican object get`: This is the client command we are utilizing. `object` because we want to interact with Pelican objects and `get` because we want to GET the file from our federation.
+- `pelican://`: This is the URL scheme for accessing this object. Pelican accepts different URL schemes which are detailed in the [client usage](./client_usage.mdx#the-different-pelican-url-schemes) page.
+- `<federation-url>`: The URL indicating the object's location.
+- `<namespace-prefix>`: The namespace at which this object is located.
+- `<path/to/federation/object>`: The file path to the object we would like to get.
+- `</local/path/to/destination>`: The local destination where we want the file to download to (can be a local or absolute path)
+
+**Try The Command Yourself**
+Now it is time for you to get your first object from a federation. Before getting your first object, be sure you are in a directory/location where you want to download this test file. To get the object, run the command below:
```

Comment:
Can they not just use something like "my-directory/downloaded-test.txt" as the destination path name if they aren't in the directory they want to be in? If they can't, possibly specify that this is a requirement for whatever reasons?

Reply:
They can, but for this example I think we should just keep it as simple/basic as possible since this is just a getting started guide. In the regular client usage and in other examples it is mentioned they can specify the path they would like to use for the destination.